ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/BulkRequest.java

```diff
@@ -56,9 +56,15 @@ private InputStream generateStream() {
         objects.setPriority(this.priority);
         objects.setWriteOptimization(this.writeOptimization);
         objects.setChunkClientProcessingOrderGuarantee(this.chunkOrdering);
-        final String xmlOutput = XmlOutput.toXml(objects, this.getCommand());
+        
+        final StringBuilder xmlOutputBuilder = new StringBuilder();
+        if (this.getCommand() == BulkCommand.GET) {
```

Comment:
Is this the right logic?  We care only about bulk PUT.  We want to add the filter for every request besides bulk put.  So we should invert this logic.